src/artemis/external_interaction/tests/test_ispyb_handler.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import MagicMock, call
+
+from artemis.external_interaction.communicator_callbacks import ISPyBHandlerCallback
+from artemis.parameters import FullParameters
+
```

Comment:
Nit: I think we could test:
* The exception on a stop w/o ids
* That ispyb only gets triggered on the correct plan